clients/common/rocblas_gentest.py

```diff
@@ -196,7 +196,7 @@ def setdefaults(test):
     # These are only for dynamic defaults
     # TODO: This should be ideally moved to YAML file, with eval'd expressions.
 
-    if all([x in test for x in ('M', 'incx', 'strideScale')]) and test['function']=='ger_strided_batched':
+    if all([x in test for x in ('M', 'incx', 'strideScale')]) and (test['function']=='ger_strided_batched' or test['function']=='scal_strided_batched'):
```

Comment:
Minor but might be worth putting the specific function name check first for readability and to short circuit as it is likely a cheaper test.

Reply:
Also, the `all()` test can be done once outside of the `if`, or as separate a nested `if` controlling multiple cases, rather than repeating it.

Reply:
Rearranged statements.